google/cloud/apigeeregistry/v1/registry_service.proto

```diff
@@ -903,6 +903,10 @@ message UpdateApiDeploymentRequest {
   // If a "*" is specified, all fields are updated, including fields that are
   // unspecified/default in the request.
   google.protobuf.FieldMask update_mask = 2;
+
+  // If set to true, and the deployment is not found, a new deployment will be created.
+  // In this situation, `update_mask` is ignored.
+  bool allow_missing = 3;
```

Comment:
Since much the deployment support was copied-and-modified from the existing spec support, it was easier to bring this over than try to remove it (and I think we want it anyway and will want to add it to our other resources)